src/pages/registration/SupportModalContent.tsx

```diff
@@ -22,11 +29,99 @@ export const SupportModalContent = ({ registration }: SupportModalContentProps)
     onError: () => dispatch(setNotification({ message: t('feedback.error.get_tickets'), variant: 'error' })),
   });
 
+  const addMessageMutation = useMutation(
+    async (data: AddTicketMessageData) => await addTicketMessage(data.ticketId, data.message),
+    {
+      onError: () => dispatch(setNotification({ message: t('feedback.error.send_message'), variant: 'error' })),
+      onSuccess: () => {
+        dispatch(setNotification({ message: t('feedback.success.send_message'), variant: 'success' }));
+        closeModal();
+      },
+    }
+  );
+
+  const createSupportTicketMutation = useMutation(
+    async (message: string) => await createTicket(registration.id, 'GeneralSupportCase', true),
+    {
+      onSuccess: (data, message) => addMessageMutation.mutate({ ticketId: data.data?.id || '', message: message }),
+      onError: () => dispatch(setNotification({ message: t('feedback.error.send_message'), variant: 'error' })),
+    }
+  );
+  const isLoading = ticketsQuery.isLoading || addMessageMutation.isLoading || createSupportTicketMutation.isLoading;
+
+  let tickets = ticketsQuery.data?.tickets ?? [];
+  tickets = tickets.filter((ticket) => ticket.type === 'GeneralSupportCase');
+  const currentSupportTicket = tickets.length > 0 ? tickets.at(tickets.length - 1) : null;
```

Comment:
Så lenge vi ikke skal bruke tickets til noe annet kan vi godt gjøre dette i en smell.
`at()` løser ganske mye native: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/at#return_the_last_value_of_an_array
less code is best code 👊 

```suggestion
  const currentSupportTicket = ticketsQuery.data?.tickets
    .filter((ticket) => ticket.type === 'GeneralSupportCase')
    .at(-1);
```

Reply:
Fikset.